Resources/Prototypes/Entities/Objects/Weapons/Guns/Revolvers/revolvers.yml

```diff
@@ -92,8 +96,11 @@
   - type: Clothing
     sprite: Objects/Weapons/Guns/Revolvers/mateba.rsi
   - type: Gun
+    fireRate: 2
     soundGunshot:
       path: /Audio/Weapons/Guns/Gunshots/mateba.ogg
+  - type: StaticPrice
+    price: 2000
```

Comment:
Also this is an ERT gun so it's not orderable and a price won't really do anything.

Reply:
> Also this is an ERT gun so it's not orderable and a price won't really do anything.

I gave the price for the future, I wanted to add it to the spawn boxes on fragments, it will give out with the same percentage as other revolvers

Reply:
ERT gear shouldn't be in salvage loot.

Reply:
> ERT gear shouldn't be in salvage loot.

ok i will change that